app/src/modules/settings/routes/translation-strings/collection.vue

```diff
@@ -29,83 +37,168 @@
 				{{ t('no_translation_string_copy') }}
 
 				<template #append>
-					<v-button @click="openTranslationStringDialog">{{ t('create_translation_string') }}</v-button>
+					<v-button @click="openTranslationStringDrawer">{{ t('create_translation_string') }}</v-button>
 				</template>
 			</v-info>
-			<v-table
-				v-else
-				:headers="tableHeaders"
-				fixed-header
-				item-key="key"
-				:items="tableItems"
-				:loading="loading"
-				@click:row="openTranslationStringDialog"
-			>
-				<template #[`item.key`]="{ item }">
-					<span class="key">
-						{{ item.key }}
-					</span>
-				</template>
-				<template #[`item.translations`]="{ item }">
-					<TranslationStringsTooltip :translations="item.translations" />
-				</template>
-			</v-table>
+			<template v-else>
+				<v-table
+					:headers="tableHeaders"
+					fixed-header
+					item-key="key"
+					:items="tableItems"
+					:loading="loading"
+					:show-resize="true"
+					@click:row="openTranslationStringDrawer"
+					@update:headers="updateHeaders"
+				>
+					<template #[`item.key`]="{ item }">
+						<span class="key">
+							{{ item.key }}
+						</span>
+					</template>
+					<template #[`item.translations`]="{ item }">
+						<TranslationStringsTooltip :translations="item.translations" />
+					</template>
+					<template #footer>
+						<div class="footer">
+							<div class="pagination">
+								<v-pagination
+									v-if="totalPages > 1"
+									:length="totalPages"
+									:total-visible="7"
+									show-first-last
+									:model-value="page"
+									@update:model-value="toPage"
+								/>
+							</div>
+							<div v-if="loading === false" class="per-page">
+								<span>{{ t('per_page') }}</span>
+								<v-select
+									:model-value="`${limit}`"
+									:items="['25', '50', '100', '250', '500', ' 1000']"
```

Comment:
I believe we need to update this to use the recent `usePageSize`, similar to this changes in Cards layout: https://github.com/directus/directus/pull/17309/files#diff-9940b7966742cae75070d5f1c877859e6712805136610a866f8db5f61fd4aacf